### PR TITLE
Simulation draft

### PR DESCRIPTION
**Pull request by MateuszKubuszok**
Version without fee nor transaction confirmation.

### CODE REVIEW

File: elm/src/test/scala/io/scalac/elm/functional/FuzzyTransactionTest.scala [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+          .fundsChangedBy(sender, -(amount + fee))
+          .fundsChangedBy(receiver, amount)
+
+        synchronized(wait(100))
+
+        performRandomTransactions(newStates, transactionsLeft -1)
+      }
+    }
+
+    val initialNode2State = nodes zip (nodes map getWalletFunds map NodeExpectedState) toMap
```
Comment (adamsmo) [inline elm/src/test/scala/io/scalac/elm/functional/FuzzyTransactionTest.scala:128]: I think we need another wait here to allow nodes to sync chain and than check state of wallets

**Comment by adamsmo** [comment]
other than mentioned looks good to me

File: elm/src/test/scala/io/scalac/elm/functional/FuzzyTransactionTest.scala (hunk 1 above)
Comment (MateuszKubuszok) [reply elm/src/test/scala/io/scalac/elm/functional/FuzzyTransactionTest.scala:128]: This is before any tranaction and right after initialization - they should have as much gold as we put in the configs, shouldn't they?

Comment (adamsmo) [reply elm/src/test/scala/io/scalac/elm/functional/FuzzyTransactionTest.scala:128]: yep, wrong place I mean between 129 and 130

Comment (rtkaczyk) [reply elm/src/test/scala/io/scalac/elm/functional/FuzzyTransactionTest.scala:128]: yeah, they need time to synchronize

Comment (adamsmo) [reply elm/src/test/scala/io/scalac/elm/functional/FuzzyTransactionTest.scala:128]: especially if `forging.delay` is set to `2s`

Comment (rtkaczyk) [reply elm/src/test/scala/io/scalac/elm/functional/FuzzyTransactionTest.scala:128]: We can fix that later

**Comment by rtkaczyk** [comment]
👍

**Pull request merged**